packages/inputs-minimal/src/radio/index.stories.jsx

```diff
@@ -0,0 +1,77 @@
+import { useState } from 'react';
+import Radio from './index';
+import styles from './index.stories.module.scss';
+
+const OPTIONS = [
+  { label: 'Option 1', value: 1 },
+  { label: 'Option 2', value: 2 },
+  { label: 'Option 3', value: 3 },
+];
+
+const OPTIONS2 = [
+  { label: 'Option 1', value: 1 },
+  { label: 'Option 2', value: 2, disabled: true },
+  { label: 'Option 3', value: 3 },
+];
+
+export const Default = () => {
+  const [value, setValue] = useState('');
+
+  return (
+    <div className={styles.container}>
+      {OPTIONS.map((option) => (
+        <Radio
+          key={option.value}
+          value={option.value}
+          label={option.label}
+          disabled={option.disabled}
+          onChange={setValue}
+          name="example1"
+          checked={value === option.value}
+        />
+      ))}
+    </div>
+  );
+};
+
+export const DefaultSelected = () => {
+  const [value, setValue] = useState('2');
```

Comment:
how does this work? '2' and 2 have different types

As a user of this library, I would think `Radio` passes the original option value in `onChange`, not a value that might have been casted to string. not sure how other libs are handling this

Reply:
it was an issue in story, radio, and select passing exact value to change handler, without type changes. so in this case there is no radio input with '2' value, so no default selected. story fixed